src/server/services/StatisticsService.ts

```diff
@@ -0,0 +1,27 @@
+import { inject, injectable } from 'inversify'
+import { IStatisticsService } from './interfaces/IStatisticsService'
+import { DependencyIds } from '../constants'
+import { IStatisticsRepository } from '../repositories/interfaces/IStatisticsRepository'
+import { GlobalStatistics } from './types'
+
+@injectable()
+export class StatisticsService implements IStatisticsService {
+  private statisticsRepository: IStatisticsRepository
+
+  public constructor(
+    @inject(DependencyIds.statisticsRepository)
+    statisticsRepository: IStatisticsRepository,
+  ) {
+    this.statisticsRepository = statisticsRepository
+  }
+
+  getGlobalStatistics: () => Promise<GlobalStatistics> = async () => {
+    const clickCount = await this.statisticsRepository.getGlobalClickCount()
+    const linkCount = await this.statisticsRepository.getGlobalLinkCount()
+    const userCount = await this.statisticsRepository.getGlobalUserCount()
```

Comment:
Previously we could retrieve all the values at once from Redis via `mget`. Can we preserve that behavior so that we don't triple the request load to Redis?